server/routerlicious/.changeset/great-plums-repair.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/server-lambdas": major
+"@fluidframework/server-lambdas-driver": major
+"@fluidframework/server-memory-orderer": major
+"@fluidframework/server-services-core": major
+"@fluidframework/server-services-ordering-rdkafka": major
+"@fluidframework/server-test-utils": major
+---
+
```

Comment:
I believe this needs a `section` defined - @tylerbutler do we have a section for service changes, or should this go in `section: fix`?

Reply:
In most circumstances you'd be absolutely right. Unfortunately, the version of build-tools in use in server right now doesn't support sections, so better just to leave them out. I'm working on getting server bumped to the latest, but it requires more build-tools changes.